pkg/users/account-details.js

```diff
@@ -240,15 +249,15 @@
                                 <FormGroup fieldId="account-real-name" hasNoPaddingTop={!superuser.allowed} label={_("Full name")}>
                                     { superuser.allowed
                                         ? <TextInput id="account-real-name"
-                                                     isDisabled={committing_real_name || account.uid == 0}
-                                                     value={edited_real_name !== null ? edited_real_name : account.gecos}
+                                                     isDisabled={comittingRealName || account.uid == 0}
+                                                     value={editedRealName !== null ? editedRealName : account.gecos}
                                                      onKeyDown={event => {
                                                          if (event.key == "Enter") {
                                                              event.target.blur();
                                                          }
                                                      }}
-                                                     onChange={(_event, value) => set_edited_real_name(value)}
-                                                     onBlur={() => change_real_name()} />
+                                                     onChange={(_event, value) => setEditedRealName(value)}
+                                                     onBlur={() => changeRealName()} />
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-7fd6cddb-20250813-150050-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)